llvm/include/llvm/CodeGen/MachineBasicBlock.h

```diff
@@ -477,7 +481,7 @@ class MachineBasicBlock
   /// Remove entry from the livein set and return iterator to the next.
   livein_iterator removeLiveIn(livein_iterator I);
 
-  std::vector<RegisterMaskPair> getLiveIns() const { return LiveIns; }
+  const std::vector<RegisterMaskPair> getLiveIns() const { return LiveIns; }
```

Comment:
Was this supposed to be?
```suggestion
  const std::vector<RegisterMaskPair> &getLiveIns() const { return LiveIns; }
```